backend/gateway/mux/mux.go

```diff
@@ -62,14 +69,51 @@ func (a *assetHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	// Serve!
 	if f, err := a.fileSystem.Open(r.URL.Path); err != nil {
+		// If not a known static asset and a asset provider is configured, try streaming from the configured provider.
+		if a.assetCfg.Provider != nil && strings.HasPrefix(r.URL.Path, "/static") {
+			// We attatch this header simply for observibility purposes.
+			// Otherwise its difficult to know if the assets are being served from the configured provider.
+			w.Header().Set("x-clutch-asset-passthrough", "true")
+
+			// TODO: handle errors
```

Comment:
I still need to handle errors some how, I haven't decided on what way would be best yet.

Reply:
We can open another PR this week to handle the error state when either both the host and the pass through fail or only the host if the pass through isn't configured.